# Operation ATTACH / DETACH

byte DETACH=0
byte ATTACH=1

byte operation

# Reference model and link to attach to
string reference_model_name
string reference_link_name

# Target model and link to attach
string target_model_name
string target_link_name



